pkg/mesh/krun.go

```diff
@@ -233,6 +236,51 @@ func (kr *KRun) InitForTD() {
 	}
 }
 
+// LoadTDBootstrapConfigurations loads optional bootstrap configs for TD from environment variables
+func (kr *KRun) LoadTDBootstrapConfigurations() error {
+	envoyPortRaw := os.Getenv("ENVOY_PORT")
```

Comment:
Not sure we want this to be configurable, can we keep it as fixed port ?
Each option has a cost ( testing, etc)

Reply:
Agreed, removed.